CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format of this document is based on [Keep a Changelog](https://keepachangelo
 
 ## [Unreleased]
 
+### Changed
+
+- `controlBoard` plugin now uses a dinamically loaded plugin for handling the coupling of the hand mk5 (see https://github.com/icub-tech-iit/ergocub-software/pull/178)
```

Comment:
I think we should separate the two things:
* the fact that we add a new feature (loading the coupling from YARP device)
* the fact that we removed support for `COUPLING::icub_hand_mk5` parameters, and that if you want to still use ergocub models you need to update to load the pass the `device couplingXCubHandMk5` parameter.

The first point is interesting for all users of gyp, the second is a strictly related to ergocub users, my suggestion:

```suggestion
### Added
- The `gazebo_yarp_controlboard` plugin gained support to load at runtime arbitrary couplings specified by a yarp device that exposes the [`yarp::dev::IJointCoupling`](https://github.com/robotology/yarp/blob/v3.9.0/src/libYARP_dev/src/yarp/dev/IJointCoupling.h#L16) interface. The name of the yarp device used to load the coupling is passed via the `device` parameter in the `COUPLING` group. For an example of PR that uses this new feature, check https://github.com/icub-tech-iit/ergocub-software/pull/178 .

### Removed

- The support for passing the `icub_hand_mk5` parameter in the `COUPLING` the `gazebo_yarp_controlboard` plugin was removed. This breaks compatibility with models contained in ergocub-software <= 0.6.0, to use those models use a newer version of ergocub-software. 
```

Reply:
fyi @Nicogene